src/main/java/net/adoptium/GreetingResource.java

```diff
@@ -1,16 +1,35 @@
 package net.adoptium;
 
 import javax.ws.rs.GET;
+import javax.ws.rs.HeaderParam;
 import javax.ws.rs.Path;
 import javax.ws.rs.Produces;
 import javax.ws.rs.core.MediaType;
+import java.util.Locale;
+import java.util.MissingResourceException;
+import java.util.ResourceBundle;
 
 @Path("/hello")
 public class GreetingResource {
 
     @GET
     @Produces(MediaType.TEXT_PLAIN)
-    public String hello() {
-        return "Hello World";
+    public String hello(@HeaderParam("Accept-Language") String lang) {
+        System.out.println("------------");
+        System.out.println(lang);
+        ResourceBundle mybundle = null;
+
+        for (Locale.LanguageRange language : Locale.LanguageRange.parse(lang)) {
```

Comment:
Quarkus provides an API for type safe message bundles for i18n: https://quarkus.io/guides/qute-reference#type-safe-message-bundles I ask myself why you decided to use raw resource bundles.